### DISCUSSION TITLE
Faking nested taxonomy terms?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5773] [General]
Hi

Is there a way to fake nested taxonomy terms?

Nesting terms isn't currently possible. So as a workaround I'm using multiple taxonomies. Which is... ok, but doing so changes the `url`

```
website.com/collection-name/taxonomy-name/term-name
website.com/products/solutions/term-name (lists all entries tagged with term)
```

Ideally, I need to use up to four taxonomies, but this would result in four different urls

```
website.com/products/solutions-level-1/term-name
website.com/products/solutions-level-2/term-name
website.com/products/solutions-level-3/term-name
website.com/products/solutions-level-4/term-name

```
Is there a way to show terms from multiple taxonomies on one url?

```
website.com/products/solutions/term-name
```
I've looked at this [Custom Taxonomy Route Workaround](https://gist.github.com/jasonvarga/4342f96b77d52fcd27dad22d88253e1f), but this looks like it only provides a way to change the name of a taxonomy route. I think I'm looking for a way to mask the route?

**Comment by afonic** [comment]
I'm not sure that understand 100% but using a custom route in web.php should do the trick. Like Jason's link but use the `Term`  facade to get the terms you need:
https://github.com/statamic/cms/blob/3.3/src/Facades/Term.php

I don't have the time to test now but something like `Term::whereInTaxonomy(['handle1', 'handle2', 'handle2'])->get()` might do the trick.
